test/e2e/clusterctl_upgrade.go

```diff
@@ -63,6 +63,7 @@ type ClusterctlUpgradeSpecInput struct {
 	InitWithBinary string
 	// InitWithProvidersContract can be used to override the INIT_WITH_PROVIDERS_CONTRACT e2e config variable with a specific
 	// provider contract to use to initialise the secondary management cluster, e.g. `v1alpha3`
+	InitWithKubernetesVersion string
```

Comment:
Can you move this so that the comment for InitWithProvidersContract is above that field?

Reply:
Done!